### PR TITLE
Added the java files

### PR DESCRIPTION
**Pull request by andreahegyesi**

### CODE REVIEW

File: src/main/java/org/example/Position.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package org.example;
+public enum Position {
+    WINNER("Winner"), RUNNER_UP("Runner-up"), THIRD_PLACE("Third Place");
```
Comment (adrianbucur83) [inline src/main/java/org/example/Position.java:3]: This class is not needed, you can just use numbers

File: src/main/java/org/example/Athlete.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    @Override
+    public String toString() {
+        return "Athlete [name= " + name + ", time= " + time + ", firstShot =" + firstShot
+                + ", secondShot= " + secondShot + ", thirdShot= " + thirdShot + "]";
+    }
+
+    public int calculatePenalty() {
+        int penalty = 0;
+        for (int i = 0; i < firstShot.length(); i++) {
```
Comment (adrianbucur83) [inline src/main/java/org/example/Athlete.java:31]: we can concatenate the strings shot1+shot2+shot3 so we don't duplicate the for loops

File: src/main/java/org/example/Biathlon.java [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+     *
+     * @param second
+     * @return
+     */
+    public static String secondsToString(int second) {
+        return second / 60 + ":" + second % 60;
+    }
+
+    public Athlete createAthlete(List<String> stringList){
+        String name = stringList.get(1);
```
Comment (adrianbucur83) [inline src/main/java/org/example/Biathlon.java:57]: altho not a requirement, some minimal validation (eg stringList.size() == x) could be used here

File: src/main/java/org/example/Biathlon.java [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+        return new Athlete(name,time, firstShot, secondShot, thirdShot);
+
+    }
+
+    public static void main(String[] args) {
+        Biathlon app = new Biathlon();
+        Scanner scanner = null;
+        try {
+            scanner = new Scanner(
+                    new File("C:\\Users\\andi1\\IdeaProjects\\SkiBiathlonStandings\\src\\main\\java\\org\\example\\results.csv"));
```
Comment (adrianbucur83) [inline src/main/java/org/example/Biathlon.java:71]: Don't use system paths, use paths from your projects' root folder. If you place the file with pom.xml in project root, you can just adress it by name: "results.csv".

File: src/main/java/org/example/Biathlon.java [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+        app.displayResults(app.athletes);
+    }
+
+    /**
+     * Sorting the athletes based on the final time result
+     *
+     * @param athletes
+     * @return sorted list of athletes
+     */
+    public List<Athlete> displayResults(List<Athlete> athletes) {
```
Comment (adrianbucur83) [inline src/main/java/org/example/Biathlon.java:93]: you have 2 methods here hidden in one, calculate results and display results.

File: src/test/java/org/example/BiathlonTest.java [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    @Test
+    void parseLine() {
+        Biathlon testApp = new Biathlon();
+        String testString = "a,b,c";
+        List<String> stringList = Arrays.asList("a", "b", "c");
+        assertEquals(stringList, testApp.parseLine(testString));
+    }
+
+    @Test
+    void displayResults() {
```
Comment (adrianbucur83) [inline src/test/java/org/example/BiathlonTest.java:19]: name your tests with this pattern: givenThreeAthletesExpectedCorrectSorting

File: src/test/java/org/example/BiathlonTest.java [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+        athleteList.add(athlete2);
+        athleteList.add(athlete3);
+        List<Athlete> athleteListCopy = athleteList;
+        assertEquals("Third name", testApp.displayResults(athleteList).get(0).name);
+        assertEquals("Second name", testApp.displayResults(athleteList).get(1).name);
+        assertEquals("First name", testApp.displayResults(athleteList).get(2).name);
+    }
+
+    @Test
+    void createAthlete() {
```
Comment (adrianbucur83) [inline src/test/java/org/example/BiathlonTest.java:35]: add test for negative flow, when input is wrong (eg null or missing data)

**Review by adrianbucur83 — COMMENTED** [review]
Very solid homework, well done. See comments on how it can be improved
